src/aura/punkCall/punkCall.css

```diff
@@ -0,0 +1,33 @@
+.THIS .beer-card{
+    background: rgb(255, 255, 255);
+    border: 1px solid rgb(221, 219, 218);
+    border-radius: .25rem;
+    background-clip: padding-box;
+    box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.10);
+    display:flex;
+    margin-top: .75rem;
+}
+.THIS .beer-card-img{
+    height:180px;
+    min-width:180px;
+    margin: 10px;
+    text-align:center;
```

Comment:
if you are using spaces between property and value be consistent and add it in every line.